### PR TITLE
更新v2中Kubernetes Nacos快速开始文档

### PR DESCRIPTION
**Pull request by InGramViking**
## What is the purpose of the change

nacos在2020年就已经[移除数据库主从配置](https://github.com/nacos-group/nacos-docker/wiki/%E7%A7%BB%E9%99%A4%E6%95%B0%E6%8D%AE%E5%BA%93%E4%B8%BB%E4%BB%8E%E9%95%9C%E5%83%8F%E9%85%8D%E7%BD%AE), 官网文档一直没有更新。
更改数据库部署文档;
更改变量名称;
修改属性名称, 添加新的必要参数;

## Brief changelog

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/nacos-group/nacos-group.github.io/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Test your code locally by running `docsite start`, and make sure it works as expected.
- [ ] Make sure no files under build directory is added.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Comment by InGramViking** [comment]
https://github.com/nacos-group/nacos-group.github.io/issues/341

**Review by KomachiSion — APPROVED** [review]

File: docs/zh-cn/v2/guide/admin/system-configurations.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ JAVA_OPT="${JAVA_OPT} -Dnacos.home=${BASE_DIR}"
 |db.url.1| 第二个数据库的URL | 字符串 | 空 | >= 0.1.0 |
 |db.user| 数据库连接的用户名 | 字符串 | 空 | >= 0.1.0 |
 |db.password| 数据库连接的密码 | 字符串 | 空 | >= 0.1.0 |
-|spring.datasource.platform|数据库类型|字符串|mysql|>=1.3.0|
+|spring.sql.init.platform|数据库类型|字符串|空|>=1.3.0|
```
Comment (KomachiSion) [inline docs/zh-cn/v2/guide/admin/system-configurations.md:50]: 这里可能需要加说明，旧的参数是完全不可用了吗？ 如果还可用最好写一下即将删除，使用xx代替。

另外默认值问题不同版本可能不一样，最好也写明。

**Review by KomachiSion — CHANGES_REQUESTED** [review]

**Comment by KomachiSion** [comment]
Relate https://github.com/alibaba/nacos/issues/9718

File: docs/zh-cn/v2/guide/admin/system-configurations.md (hunk 1 above)
Comment (InGramViking) [reply docs/zh-cn/v2/guide/admin/system-configurations.md:50]: 这里我很纠结怎么改，建议还是committer改吧，我在alibaba/nacos仓库 v2.2.1-RC中 https://github.com/alibaba/nacos/blob/54310077f0fec96e447548202788a44145f60aae/distribution/conf/application.properties#L35 看到的注释依然是`spring.datasource.platform=mysql`, 但是在nacos-group/nacos-docker仓库 v2.2.0 https://github.com/nacos-group/nacos-docker/blob/05086aec39ce57d8c0bbf0845c3600c44f0663a4/build/conf/application.properties#L13 的注释就已经说配置`spring.datasource.platform`被弃用了。
这个配置我是从nacos-group/nacos-docker仓库 v2.2.0开始看到的，v2.1.2还没有注释说明被弃用。
可能committer理解的更好点。

默认值同样, 根据 paderlol 的说法https://github.com/alibaba/nacos/issues/9718#issuecomment-1365668060 默认为空则默认为嵌入式, 我理解的是适用于v2.2.0版本开始(v2.2.0开始出现的弃用注释)。

Comment (InGramViking) [reply docs/zh-cn/v2/guide/admin/system-configurations.md:50]: 本次PR主要想尽快更新官网对于Kubernetes Nacos的部署文档，实在是太老了，这两天又看到有人遇到文档问题https://github.com/nacos-group/nacos-k8s/issues/384
这个文件的提交我可以暂时关闭, 建议还是committer来改，更严谨，我只从运维使用k8s/docker部署的方向来写的。

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**